mffpy/xml_files.py

```diff
@@ -841,6 +841,7 @@ class Categories(XML):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self._segment_converter = {
+            'name': lambda e: None if e is None else str(e.text),
```

Comment:
I think it would be good to have an explicit separation of required and optional elements here.